llama_stack/providers/inline/telemetry/meta_reference/sqlite_span_processor.py

```diff
@@ -153,6 +154,9 @@ def on_end(self, span: Span):
             )
 
             for event in span.events:
+                name = event.name
```

Comment:
probably the wrong fix. we need to make sure the value getting generated is correct.